mmseg/apis/inference.py

```diff
@@ -134,3 +137,6 @@ def show_result_pyplot(model,
     plt.title(title)
     plt.tight_layout()
     plt.show(block=block)
+    if out_file is not None:
+        plt.savefig(out_file)
+    plt.cla()
```

Comment:
I suggest use `mmcv.imwrite`, as the default channel order of plt is RGB, and now the order of img is BGR, and the backend of `mmcv.imwrite` is cv2 defualting to BGR 
https://github.com/open-mmlab/mmcv/blob/3ede2789cc44d827d4687995707c7c6db74bb03a/mmcv/image/io.py#L312